x-pack/plugins/security/public/views/management/edit_role/components/privileges/kibana/__snapshots__/space_aware_privilege_form.test.tsx.snap

```diff
@@ -100,12 +100,12 @@ exports[`<SpaceAwarePrivilegeForm> renders without crashing 1`] = `
       size="s"
     >
       <p>
-        Customize permission levels per space. If a space is not customized, its permissions will default to the minimum privilege specified above.
-      </p>
-      <p>
-        You can bulk-create space privileges though they will be saved individually upon saving the role.
+        Grant more privileges on a per space basis. For example, if users have read privileges in all spaces, you can grant all privileges to an individual space.
       </p>
```

Comment:
I got some feedback from another writer that the second line here is hard to parse.  Here is an edited version:

For example, if the privileges are **read** for all spaces, you can set the privileges to **all** for an individual space.

Other than that change, the text LGTM.